globus_cli/commands/login.py

```diff
@@ -13,6 +13,23 @@
     internal_auth_client, write_option)
 
 
+_LOGIN_EPILOG = ("""\
+
+SUCCESS
+You have logged in to the Globus CLI as {}
+
+A set of valid tokens for accessing Globus have been saved for you in
+~/.globus.cfg and are valid indefinitely unless revoked.
```

Comment:
Fair points. I wanted people to know that their tokens are being stored on the local filesystem though, since people could be easily confused by going out to a remote host, running the CLI, and then authenticating through a local web browser.

What about "A set of valid tokens for accessing Globus have been saved for you in your home directory."?